engine/src/main/java/org/entando/entando/aps/system/services/pagemodel/api/ApiPageModelInterface.java

```diff
@@ -163,8 +158,8 @@ public String getApiResourceUrl(Object object, String applicationBaseUrl, String
 		}
 		PageModel pageModel = (PageModel) object;
 		StringBuilder stringBuilder = new StringBuilder(applicationBaseUrl);
-		stringBuilder.append("api/rs/").append(langCode).append("/core/pageModel");//?code=").append(pageModel.getCode());
-		if (null == mediaType || mediaType.equals(MediaType.APPLICATION_XML_TYPE)) {
+		stringBuilder.append("api/legacy/").append(langCode).append("/core/pageModel");
```

Comment:
can we use a constant for `legacy` or `/core/pageModel` (shared with controller configuration i mean ...) ?

Reply:
done, as above